packages/db/utils.js

```diff
@@ -122,3 +122,13 @@ export function parseTextToNumber (numberText) {
   }
   return num
 }
+
+/**
+ * @param {Number} num
+ */
+export function safeNumber (num) {
```

Comment:
This is not parsing anything - just ensuring it's within the safe integer range.

Maybe `validateSafeInteger`?

Reply:
what I meant is 
```
export function parseTextToNumber (numberText) {
  const num = Number(numberText)
  validateSafeInteger(num)
  return num
}

/**
 * @param {Number} num
 */
export function validateSafeInteger (num) {
  if (!Number.isSafeInteger(num)) {
    throw new Error('Invalid integer number.')
  }
  return num
}
```

to avoid duplicating the validation in 2 functions... 

Really minor ;)